packages/plugin-xml/src/Attributes.ts

```diff
@@ -0,0 +1,45 @@
+import { Modifier } from '../../core/src/Modifier';
+import { deepEqualObjects } from '../../utils/src/utils';
+
+export class Attributes extends Modifier {
+    value: Record<string, string> = {};
+    constructor(valueOrAttributes?: Record<string, string> | Attributes) {
```

Comment:
I think I would just have named this variable `attributes`. I mean, it does represent "attributes" anyway, no matter the "encoding" per se.